nucypher/cli/commands/worklock.py

```diff
@@ -364,35 +373,35 @@ def enable_claiming(general_config, worklock_options, force, hw_wallet, gas_limi
                           f" using {worklock_options.bidder_address}?", abort=True)
```

Comment:
I guess this also will be part of `literature`?

Reply:
Hmm - Perhaps - although right now literature only defines strings used in emitters, not exceptions, confirmations, or prompts.

Reply:
But I see a lot of strings for confirmations even in this method ('CONFIRM_BID_VERIFICATION' for example), what is the differnece?

Reply:
Violating my own rule? :sweat_smile: 

I'd actually like to consume the rest of the prompts and confirmations too.